helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -31,7 +31,7 @@ spec:
       annotations:
         description: '{{`Prometheus can''t communicate with Remote Storage API at {{ $labels.url }}.`}}'
         opsrecipe: prometheus-cant-communicate-with-remote-storage-api/
-      expr: rate(prometheus_remote_storage_samples_failed_total[1h]) > 0.10 or rate(prometheus_remote_storage_samples_total[1h]) == 0 or rate(prometheus_remote_storage_metadata_retried_total[5m]) > 0
+      expr: rate(prometheus_remote_storage_samples_failed_total{instance!="prometheus-agent"}[1h]) > 0.1 or rate(prometheus_remote_storage_samples_total{instance!="prometheus-agent"}[1h]) == 0 or rate(prometheus_remote_storage_metadata_retried_total{instance!="prometheus-agent"}[5m]) > 0
```

Comment:
I'm not sure I would restrict this alert as this could still happen if the agent is up. We might want to inhibit it if the other alert triggers though

Reply:
I do think we need to be alerted

Reply:
I think its good that we exclude the prometheus agent from this alert, this alert serves a different purprose and is less critical.

Reply:
Then maybe we need to create another alert with an other severity for the agent

Reply:
Because we need to know when the agent is down but also when the samples are failing to come

Reply:
@mcbenjemaa why is this resolved? I would still create another alert if we get failed samples over time for the agent. We can create a specifuc remotewrite alert for the agent sure

Reply:
for 1h ? 
like the one above